### ISSUE TITLE
Detect and prevent spelling errors in documentation

### ISSUE THREAD

**Issue by nvzard** [coala/documentation#478]
Use a coala bear to detect spelling mistakes in current documentation , and prevent them from re-occurring by improving the `.coafile`.

**Comment by Makman2** [comment]
In that case we should also have a bear that detects spelling mistakes, so we have at least some spell checks ;)

**Comment by nvzard** [comment]
Yeah, that's a good idea. Should I propose a spellchecker bear? Please mark this issue as initiative/gci

**Comment by satwikkansal** [comment]
> we should also have a bear that detects spelling mistakes

Yeah, otherwise manually going through the documentation to detect mistakes would be a boring task I feel.

**Comment by Makman2** [comment]
@nvzard Some bears for markdown and others do have spellcheck modules, so we might not have to write a new bear :+1:

**Comment by jayvdb** [comment]
This is about using a spelling checking bear.  We have them.

**Comment by newbazz** [comment]
![image](https://user-images.githubusercontent.com/25884863/32939176-b42a9604-cba4-11e7-8c10-de13194fbeb8.png)
Shouldnt it be `on the internet`.

**Comment by Makman2** [comment]
I think both is okay^^

**Comment by jayvdb** [comment]
Problems with implementation at https://github.com/coala/coala/pull/4937 and a possible solution at https://gitlab.com/gitmate/open-source/gitmate-2/issues/240
